config/locales/en/brexit_checker/breadcrumbs.yml

```diff
@@ -3,6 +3,7 @@ en:
     breadcrumbs:
       home: Home
       brexit-home: Brexit
-      brexit-check: How to get ready
+      brexit-check: "Get ready for Brexit: Your results"
       questions: Questions
       results: Results
+  
```

Comment:
I think you need to configure your IDE to add a newline to the end of new files automatically - I can see a couple of warnings about it on the new files in this PR (e.g: this one)

Reply:
argh, thought that was on by default. Enabled now. Thanks!